CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
     Click to see more.
   </summary>
 
+### Major
+* Video: Add new Video component to Gestalt (#150)
```

Comment:
@pfarejowicz Even though we're adding a new component, it should be `Minor`, since we're still on `0.x.x` releases

Reply:
Too excited about adding videos haha